scripts/release-steps.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# Exit on error and output commands.
+set -ex
+
+NEXT_VERSION=$1
+if [ "$NEXT_VERSION" = "" ]; then
+	echo "Error: Version not set!"
+	exit
+fi
+
+# Fix permissions for ssh config and keys.
+chown -R root:root /root/.ssh
+export GIT_SSH_COMMAND="ssh -i /root/.ssh/id_rsa"
+
+# Create working directory and checkout Sensei.
+mkdir /release && cd /release
+git clone git@github.com:Automattic/sensei.git
+cd /release/sensei
+
+# Install dependencies.
+composer install && npm install
+
+# Disable commit signing.
+git config --global commit.gpgsign false
+
+# Create the release branch.
+git switch -c "release/$NEXT_VERSION"
+
+echo "Replace next version tag"
+./scripts/replace-next-version-tag.sh $NEXT_VERSION
+git add .
+git commit -m 'Replace next version tag'
+
+echo "Changlogger write"
+composer exec -- changelogger write
+git add .
+git commit -m 'Update chaneglog'
+
+echo "Build translations"
+npm run build:assets && wp i18n make-pot --exclude=build,lib,vendor,node_modules,assets/vendor --headers='{"Last-Translator":null,"Language-Team":null,"Report-Msgid-Bugs-To":"https://wordpress.org/support/plugin/sensei-lms"}' . lang/sensei-lms.pot --allow-root
+git add .
```

Comment:
Why we need to add 2 commits?

Reply:
I followed the conventional approach I saw previously in our releases.
That follows also our desire to have atomic commits.